frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -80,10 +83,10 @@ class HtmlDocument extends React.Component {
     ])
 
     props.registerLiveMessageHandlerList([
-      { entityType: TLM_ET.CONTENT, coreEntityType: TLM_CET.MODIFIED, handler: this.handleContentModified },
-      { entityType: TLM_ET.CONTENT, coreEntityType: TLM_CET.CREATED, handler: this.handleContentCreated }
-      // { entityType: TLM_ET.CONTENT, coreEntityType: TLM_CET.DELETED, handler: this.handleContentDeleted }
-      // { entityType: TLM_ET.CONTENT, coreEntityType: TLM_CET.UNDELETED, handler: this.handleContentUndeleted }
+      { entityType: TLM_ET.CONTENT, coreEntityType: TLM_CET.MODIFIED, optionalSubType: CONTENT_TYPE.HTML_DOCUMENT, handler: this.handleContentModified },
```

Comment:
as seen with @metzgegu , in his PR he created a new constant `const TLM_SUB_TYPE = { ...CONTENT_TYPE }`
And I think it's better than using `CONTENT_TYPE`.
I told him to refactor my part in frontend in his PR already. So, you can just do as he did [here](https://github.com/tracim/tracim/pull/3083/files#diff-f848bd06614f11b92f9fcb127affcb39R14) please

Reply:
Changed